mpl_data_cast/gui/preferences.py

```diff
@@ -68,6 +77,8 @@ def on_settings_ok(self):
                         value = str(pathlib.Path.home())
             elif isinstance(widget, QtWidgets.QSpinBox):
                 value = int(widget.value())
+            elif isinstance(widget, QtWidgets.QComboBox):
+                value = widget.currentText()
```

Comment:
Instead of `currentText`, use `currentData`, which is not subject to internationalization (not that we are ever going to translate this thing, but it's cleaner in any case).